dev/io.openliberty.microprofile.openapi.internal.common/resources/io/openliberty/microprofile/openapi/internal/resources/OpenAPI.nlsprops

```diff
@@ -63,4 +63,29 @@ OPENAPI_APPLICATION_PROCESSED.useraction=No action is required.
 
 OPENAPI_APPLICATION_PROCESSING_ERROR=CWWKO1661E: An error occurred when processing application {0} and an OpenAPI document was not produced. The error was: {1}.
 OPENAPI_APPLICATION_PROCESSING_ERROR.explanation=A runtime error occurred when processing the application.
-OPENAPI_APPLICATION_PROCESSING_ERROR.useraction=Review the FFDC logs and exception text to identify the problem.
\ No newline at end of file
+OPENAPI_APPLICATION_PROCESSING_ERROR.useraction=Review the FFDC logs and exception text to identify the problem.
+
+OPENAPI_MERGE_PROBLEMS_CWWKO1662W=CWWKO1662W: The following problems occurred while merging OpenAPI documents: {0}
+OPENAPI_MERGE_PROBLEMS_CWWKO1662W.explanation=The OpenAPI documentation from multiple web modules could not be successfully merged. If the reported problems are not fixed, the endpoints from the reported modules will not be included in the generated OpenAPI document.
+OPENAPI_MERGE_PROBLEMS_CWWKO1662W.useraction=Review the merge problems to identify why the generated OpenAPI documents could not be merged.
+
+OPENAPI_MERGE_PROBLEM_PATH_CLASH=Path {0} from module {1} clashes with a path from module {2}. {1} will not be merged.
```

Comment:
I think this message still needs it's own code

Reply:
Even though it's included as part of another message? It's one of the problems which will be listed under CWWKO1662W (actually I think it's currently the only problem).

Reply:
@OpenLiberty/message-reviewer we need some guidance here. Even though this message will be nested inside another one, I think it still needs a NLS code?

Reply:
In message strings with a variable, please try to use "The _variable_ noun" format.

Suggested change to message:

The {0} path from the {1} module clashes with a path from the {2} module. {1} cannot be merged.

Reply:
I've changed this as recommended but I'm not sure about it. I'm happy with "The _variable_ noun" where the variable is a name, but it seems odd when the variable is a value (as it is here).

Example with placeholders filled in:
> The /product/{id} path from the foo/api module clashes with a path from the bar/products module. foo/api cannot be merged.

Reply:
The "The _variable_ noun" example looks okay.

Reply:
Cool, that's what we now have. Thanks :)